backup.rb

```diff
@@ -18,13 +18,25 @@
 time = Time.now.strftime("%Y-%m-%d")
 filename = "backup.#{Time.now.to_i}.#{time}.sql.dump"
 
-`pg_dump -Fc --username=#{pg_user} --no-password --host #{pg_host} --port #{pg_port} #{pg_database} > #{filename}`
+if pg_user && pg_host
+  `pg_dump -Fc --username=#{pg_user} --no-password --host #{pg_host} --port #{pg_port} #{pg_database} > #{filename}`
+elsif pg_host
+  `pg_dump -Fc --host #{pg_host} --port #{pg_port} #{pg_database} > #{filename}`
+else
+  `pg_dump -Fc #{pg_database} > #{filename}`
+end
 
 # verify file exists and file size is > 0 bytes
 unless File.exists?(filename) && File.new(filename).size > 0
   raise "Database was not backed up"
 end
 
+if ENV["ENCRYPT_TO"]
```

Comment:
I found a few things that show data being pipped into `gpg --encrypt -r identity-key`. I don't have gpg keys set up to play around with it so I don't know for sure.